rollup.globals.json

```diff
@@ -13,5 +13,6 @@
 	"@tradeshift/elements.progress-bar": "ts.elements.progress-bar",
 	"@tradeshift/elements.root": "ts.elements.root",
 	"@tradeshift/elements.tooltip": "ts.elements.tooltip",
-	"@tradeshift/elements.typography": "ts.elements.typography"
+	"@tradeshift/elements.typography": "ts.elements.typography",
+	"@tradeshift/elements.note": "ts.elements.note"
```

Comment:
Make them sort by characters

Reply:
🤔 Why the `"npx sort-package-json"` didn't work? 
I'll check it!